uav_cloud: uav_cloud.ply
mls_cloud_folder: tiles
offset: [-399000.00,-6786000.00,0.0]
mls_registered_cloud_folder: tiles_transformed
optimized_cloud_output_folder: tiles_optimized
debug: False
save_pose_graph: True
log_level: DEBUG
load_clouds: True
tiles: True
icp_fitness_score_threshold: 0.85
noise_matrix: [1e+06, 0, 0, 0, 0, 0, 1e+06, 0, 0, 0, 0, 1e+06, 0, 0, 0, 10000, 0, 0, 10000, 0, 10000]
